src/System.Text.Formatting.Globalization/src/project.lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "locked": true,
+  "locked": false,
```

Comment:
I just changed the project.json file to not use wildcards, and so I think nuget changes locked to false as there is nothing to ever upgrade to.